### PR TITLE
unexpected error when using vlan list with --limit 0 or 999 and a lar…

### PR DESCRIPTION
**Pull request by caberos**
[#issue1844](https://github.com/softlayer/softlayer-python/issues/1844)

image

### CODE REVIEW

**Review by edsonarios — APPROVED** [review]
Looks good to me

File: SoftLayer/CLI/vlan/list.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ def cli(env, sortby, datacenter, number, name, limit):
             vlan.get('vlanNumber'),
             vlan.get('fullyQualifiedName'),
             vlan.get('name') or formatting.blank(),
-            vlan.get('networkSpace').capitalize(),
+            vlan.get('networkSpace').capitalize() or formatting.blank(),
```
Comment (allmightyspiff) [inline SoftLayer/CLI/vlan/list.py:68]: Apparently Vlans with a null `networkSpace` are Direct Link vlans, so lets label them as such.

```suggestion
            vlan.get('networkSpace', 'DIRECT LINK').capitalize() ,
```

Also, I think `vlan.get('networkSpace').capitalize() or formatting.blank()` will still error on an empty `networkSpace` property. Setting a default for `vlan.get` should fix that.

**Review by allmightyspiff — CHANGES_REQUESTED** [review]
Change the default response for a null `networkSpace` to `Direct Link`
Add a unit test for getting a vlan with an empty networkSpace property as well.

Thanks

**Review by allmightyspiff — APPROVED** [review]
